docs/configuration_hotfixes.md

```diff
@@ -0,0 +1,203 @@
+
+## Additional kernel arguments
+
+When creating a [performance profile CR](../deploy/crds/performance.openshift.io_v1alpha1_performanceprofile_cr.yaml) , a default set of kernel arguments are created from the [openshift-performance](https://github.com/openshift-kni/performance-addon-operators/blob/master/build/assets/tuned/openshift-node-performance) base profile in addition to tuned generated argument and include:
+
+`nohz=on rcu_nocbs=<isolated_cores> tuned.non_isolcpus=<not_isolated_cpumask> intel_pstate=disable nosoftlockup tsc=nowatchdog intel_iommu=on iommu=pt systemd.cpu_affinity=<not_isolated_cores> isolcpus=<isolated_cores> default_hugepagesz=<DefaultHugepagesSize> hugepagesz=<hugepages_size> hugepages=<hugepages_>`
+
+> Note: isolcpus is added only when [balanceIsolated](performance_profile.md#cpu) is disabled.
+
+Additional kernel arguments could be added in the performance profile CR using the `additionalKernelArgs` field:
+
+```yaml
+apiVersion: performance.openshift.io/v1alpha1
+kind: PerformanceProfile
+metadata:
+  name: example-performanceprofile
+spec:
+  additionalKernelArgs:
+  - "nmi_watchdog=0"
+  - "audit=0"
+  - "mce=off"
+  - "processor.max_cstate=1"
+  - "idle=poll"
+  - "intel_idle.max_cstate=0"  
+...
+```
+
+> Note: These arguments will be added on top of the default arguments mentioned above. Editing these additional arguments could be done when editing the CR. 
+
+## Custom tunings 
+
+To perform hotfixes on top of the tuned [openshift-performance](https://github.com/openshift-kni/performance-addon-operators/blob/master/build/assets/tuned/openshift-node-performance) base profile, a tuned custom profile (A child profile) will be used to apply the desired changes.
+This profile will inherit the base tuned profile and override its fields where needed. 
+
+
+### Getting the current deployed tuned profile
+
+In order to apply changes we will need to get the name of the deployed tuned profile that was generated by the performance addon operator:
+
+```
+#oc describe performanceprofile <profile name> | grep Tuned
+Tuned:  <tuned namespace>/<tuned name>
+
+```
+Any tuned profile created for custom tunings will need to inherit from this tuned profile: 
+
+`include=<tuned name>`
+
+for example:
+
+```yaml
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: configuration-hotfixes
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+  - data: |
+      [main]
+      summary=...
+      # override performance addons generated tuned profile
+      include=openshift-node-performance-manual
```

Comment:
Please link to Tuned or tuned documentation from somewhere around here. So the users know how to prepare the profile.

Reply:
added link